src/main/java/uk/gov/hmcts/ccd/domain/enablingcondition/jexl/JexlEnablingConditionParser.java

```diff
@@ -1,4 +1,4 @@
-package uk.gov.hmcts.ccd.domain.casestate.jexl;
```

Comment:
are we able to support enabling condition on null fields? And on blank String fields? what is the notation that we need to use? same for both?
for example something like this what would match:
`claimantIndType.claimant_title1 ==""`

Reply:
actually can we have a unite test for the scenarios above?

Reply:
Null we can't check, as we are using Map, which not allowed to save nulls